### ISSUE TITLE
something is wrong with adp subset by time

### ISSUE THREAD

**Issue by dankelley** [dankelley/oce#2160]
I won't bother explaining.  Only Clark and I have the RDA file, anyway.  Note that I am loading up a file built over a decade ago, so the most likely problem is that something in the object structure has changed.  But it's tricky reading the data (for reasons Clark will remember -- it's a split-up file) and so it might be difficult for me to reconstruct the RDA file.  I'll see if there is a way to fix this with the dataset as is.  If that fails, I'll consider revisiting the task of building that RDA.  Or I may just drop this.

<img width="742" alt="Screenshot 2023-09-30 at 7 58 07 AM" src="https://github.com/dankelley/oce/assets/99469/53d6338f-f6e2-42ef-867a-00001d7e5cdd">

**Comment by dankelley** [comment]
This is odd.  I'm inserting code to trace the problem and getting an odd message about promises.  Maybe that's just a thing of how browser is working, but around here is where the error is occuring.
```R
Browse[1]> ylim
[1] 0.9 9.9
Browse[1]> sort(ylim)
[1] 0.9 9.9
Warning messages:
1: In get(object, envir = currentEnv, inherits = TRUE) :
  restarting interrupted promise evaluation
2: In get(object, envir = currentEnv, inherits = TRUE) :
  restarting interrupted promise evaluation
```

**Comment by dankelley** [comment]
Um, I am testing `flipy`.  It is (now) supposed to be a logical.  But it is coming out as NA instead.  I will trace why that's the case and fix it, but there is something I hadn't realized before:
```R
> is.logical(NA)
[1] TRUE
```

That just seems weird to me.  If I use `is.numeric(NA)` I get FALSE.

Why is this relevant?  Because `imagep()` checks `flipy` and insists that it be logical.  Frankly, I wonder if this NA being logical is a bug in R, but anyway I will ensure that it can work either way.

**Comment by dankelley** [comment]
The problem is that all the orientations in this RDA are NA.  We have some weird things with these files, so I am not going to investigate further why that is the case.  (It's been a decade since the data were examined closely.). But I will alter oce so that this edge case can produce a graph, without dying.

**Comment by dankelley** [comment]
Fixed in "develop" branch, commit c4ad0547b6c7f6dd3cc536b7536e07ff0bb88f7d
